--- gcc/config/darwin.c	2015/07/09 17:50:58	225623
+++ gcc/config/darwin.c	2015/07/09 17:56:23	225624
@@ -1232,6 +1232,11 @@
 void
 darwin_mark_decl_preserved (const char *name)
 {
+  /* Actually we shouldn't mark any local symbol this way, but for now
+     this only happens with ObjC meta-data.  */
+  if (darwin_label_is_anonymous_local_objc_name (name))
+    return;
+
   fprintf (asm_out_file, "\t.no_dead_strip ");
   assemble_name (asm_out_file, name);
   fputc ('\n', asm_out_file);
